docs/content/next-pwa/getting-started.mdx

```diff
@@ -378,7 +378,7 @@ export default function App({ Component, pageProps }: AppProps) {
 
 ## Tips
 
-- [You may want to ask user to reload when a new service worker is installed](https://github.com/DuCanhGH/next-pwa/tree/master/examples/lifecycle/pages/index.tsx#L20-L41)
+- [You may want to ask user to reload when a new service worker is installed](https://github.com/DuCanhGH/next-pwa/tree/master/examples/lifecycle/components/PWALifecycle.tsx#L20-L40)
```

Comment:
I tried looking for when it changed from the pages to the new app router of next js 

and i found a very similiar part and changed the link for that